packages/editor/templates/Crypto Pizzas/contracts/CryptoPizza.sol

```diff
@@ -1,89 +1,108 @@
+// Specifies that the source code is for a version
+// of Solidity greater than 0.5.10
 pragma solidity ^0.5.10;
 
-import "../node_modules/openzeppelin-solidity/contracts/token/ERC721/IERC721.sol";
-import "../node_modules/openzeppelin-solidity/contracts/token/ERC721/IERC721Receiver.sol";
-import "../node_modules/openzeppelin-solidity/contracts/introspection/ERC165.sol";
-import "../node_modules/openzeppelin-solidity/contracts/math/SafeMath.sol";
-
+// Imports symbols from other files into the current contract.
+// In this case, a series of helper contracts from OpenZeppelin.
+// https://solidity.readthedocs.io/en/latest/layout-of-source-files.html#importing-other-source-files
+import "../node_modules/@openzeppelin/contracts/token/ERC721/IERC721.sol";
+import "../node_modules/@openzeppelin/contracts/token/ERC721/IERC721Receiver.sol";
+import "../node_modules/@openzeppelin/contracts/introspection/ERC165.sol";
+import "../node_modules/@openzeppelin/contracts/math/SafeMath.sol";
+
+// A contract is a collection of functions and data (its state)
+// that resides at a specific address on the Ethereum blockchain.
+// "is" keyword derives functions and keywords from the two external contracts
 contract CryptoPizza is IERC721, ERC165 {
-
     // Use Open Zeppelin's SafeMath library to perform arithmetic operations safely.
     using SafeMath for uint256;
 
-    /*
-        A Pizza has only 5 parts that can change according to its DNA.
-        When we generate DNA, we get a 10 digit long number, and each pair corresponds to a specific ingredient.
-
-        E.g. DNA 5142446803 - 51_Basis/42_Cheeses/44_Meats/68_Spices/03_Vegetables
-    */
-    uint constant dnaDigits = 10;
-    uint constant dnaModulus = 10 ** dnaDigits;
-
-    // Equals to `bytes4(keccak256("onERC721Received(address,address,uint256,bytes)"))`
-    // which can be also obtained as `IERC721Receiver(0).onERC721Received.selector`
+    // Constant variables in Solidity are similar to other languages
+    // but are not allowed to access storage, blockchain, or execution data,
```

Comment:
What do you mean that a constant variable is "not allowed to access storage"? These constant variables _are_ storage variables, aren't they?